### PR TITLE
S01 us4 prestations html css

### PR DESCRIPTION
**Pull request by Erdnaxela45**

### CODE REVIEW

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

File: src/View/Home/_prestations.html.twig [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<section class="prestations">
+    <div class="container">
+        <div class="prestations-heading">
+          <h2>Prestations</h2>
+          <p>Toutes les formules sont sur rendez-vous.</p>
+          <p class="schedule">Du lundi au vendredi de 10h à 19h - Le samedi de 11h à 16h - Fermé le dimanche et les jours fériés</p>
+        </div>
```
Comment (LucyMerlier) [inline src/View/Home/_prestations.html.twig:7]: indentation?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
indentation

**Review by Mcbess-san — APPROVED** [review]

**Review by Mcbess-san — APPROVED** [review]

**Review by Mcbess-san — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**